client/my-sites/stats/feedback/modal/index.tsx

```diff
@@ -56,7 +85,14 @@ const FeedbackModal: React.FC< ModalProps > = ( { isOpen, onClose } ) => {
 					onChange={ setContent }
 				/>
 				<div className="stats-feedback-modal__button">
-					<StatsButton primary>{ translate( 'Submit' ) }</StatsButton>
+					<StatsButton
+						primary
+						onClick={ onFormSubmit }
+						busy={ isSubmittingFeedback || isSubmissionSuccessful }
```

Comment:
I'm wondering whether we should make the button busy when `isSubmissionSuccessful`? 🤔

Reply:
Actually, that would be a tricky moment after submitting the form. Users can click the `Submit` button again before [the successful submission closes the modal](https://github.com/Automattic/wp-calypso/pull/94030/files#diff-5344886d7efa5d26bdc69d9b838cc177d4ce72a7f7bd49f3b9fd590abb5a3776R48-R56). As the modal should be closed after the successful submission, I used it to disable the submit button as well.

Reply:
ah that sounds reasonable. I thought if we are already disabling it, it might not be necessary to mark it as busy 🤔

Reply:
> I thought if we are already disabling it, it might not be necessary to mark it as busy 🤔

I know what you meant now. 😄

Reply:
Updated in [070a23a](https://github.com/Automattic/wp-calypso/pull/94030/commits/070a23ad6ae4f31670e9d749d4bb79898af7591a).

Reply:
Updated in [070a23a](https://github.com/Automattic/wp-calypso/pull/94030/commits/070a23ad6ae4f31670e9d749d4bb79898af7591a). cc @kangzj